### PR TITLE
MBS-13820: Added links to Mastodon and Bluesky in the footer

### PR DESCRIPTION
**Pull request by Suvid-Singhal**
# Problem
[MBS-13820](https://tickets.metabrainz.org/browse/MBS-13820)

# Solution
- Removed the Twitter URL from the footer
- Added links to Mastodon and Bluesky in the footer

# Testing
These were just minor cosmetic changes and they work fine on my test environment.

### CODE REVIEW

**Comment by reosarevok** [comment]
@brainzbot, ok to test

**Review by yvanzo — APPROVED** [review]
Thank you @Suvid-Singhal for this contribution.

I just reformatted your pull request’s title to make it readable by our ticket-updater bot. See:
- https://github.com/metabrainz/musicbrainz-server/blob/master/CONTRIBUTING.md
- https://github.com/metabrainz/guidelines/blob/master/GitHub.md

@reosarevok, I guess that we will wait for the decision to be taken in next Monday meeting before merging this change?

**Comment by reosarevok** [comment]
@yvanzo: I don't think we have to - Rob already said he wanted to do this. My meeting question is not "should these be the priority" but "are we planning to quit Twitter entirely" :)

**Review by reosarevok — APPROVED** [review]

File: root/account/Register.js
Comment (mwiencek) [inline root/account/Register.js:81]: Very annoying question: any reason Bluesky is listed here first but second in the footer?

Comment (reosarevok) [reply root/account/Register.js:81]: Not really - but given they're supposed to be at the same level, maybe it's good to keep them wondering!

**Pull request merged**
